cmd/network_update.go

```diff
@@ -97,6 +99,8 @@ func init() {
 	networkUpdateCmd.RegisterFlagCompletionFunc("ipv6-address", cobra.NoFileCompletions)
 	networkUpdateCmd.RegisterFlagCompletionFunc("ipv6-gateway", cobra.NoFileCompletions)
 	networkUpdateCmd.RegisterFlagCompletionFunc("ipv6-method", ipMethodCompletions)
+	networkUpdateCmd.RegisterFlagCompletionFunc("ipv6-addr_gen_mode", ipAddrGenModeCompletions)
```

Comment:
Is there a reason this is underscore instead of dashes? :thinking: 

```suggestion
	networkUpdateCmd.RegisterFlagCompletionFunc("ipv6-addr-gen-mode", ipAddrGenModeCompletions)
```

Reply:
Just a copy pasta. 😉